spec/models/manageiq/providers/ansible_tower/automation_manager/refresher_spec.rb

```diff
@@ -1,14 +1,39 @@
 describe ManageIQ::Providers::AnsibleTower::AutomationManager::Refresher do
-  let(:auth)                    { FactoryGirl.create(:authentication) }
+  # To re-record cassettes or to add cassettes you can add another inner `VCR.use_cassette` block to the
+  # 'will perform a full refresh' example. When running specs, new requests are recorded to the innermost cassette and
+  # can be played back from  any level of nesting (it tries the innermost cassette first, then searches up the parent
+  # chain) - http://stackoverflow.com/a/13425826
+  #
+  # To add a new cassette
+  #   * add another block (innermost) with an empty cassette
+  #   * change existing cassettes to use your working credentials
+  #   * run the specs to create a new cassette
+  #   * change new and existing cassettes to use default credentials
+  #
+  # To re-record a cassette
+  #   * temporarily make the cassette the innermost one (see above about recording)
+  #   * rm cassette ; run specs
+  #   * change back the order of cassettes
+  #
+  # To change credentials in cassettes:
+  # replace with defaults - before committing
+  # ruby -pi -e 'gsub /dev-ansible-tower3.cloudforms.lab.eng.rdu2.redhat.com/, "dev-ansible-tower2.example.com"; gsub /admin:smartvm/, "testuser:secret"' spec/vcr_cassettes/manageiq/providers/ansible_tower/automation_manager/*.yml
+  # replace with your working credentials
+  # ruby -pi -e 'gsub /dev-ansible-tower2.example.com/, "dev-ansible-tower3.cloudforms.lab.eng.rdu2.redhat.com"; gsub /testuser:secret/, "admin:smartvm"' spec/vcr_cassettes/manageiq/providers/ansible_tower/automation_manager/*.yml
+
+  let(:tower_url) { ENV['TOWER_URL'] || "https://dev-ansible-tower2.example.com/api/v1/" }
```

Comment:
note for later, would be nice to use application secrets like Azure does https://github.com/manageiq/manageiq-providers-azure/blob/0e30064673b3fed4bfe737ec28fe79279c2a27f0/spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb#L9-L9

I've planned to use this in AWS for ages :-)